common/scala/src/main/resources/application.conf

```diff
@@ -549,6 +549,11 @@ whisk {
         # See https://github.com/apache/openwhisk/pull/4284
         # for details
         require-api-key-annotation = true
+
+        # Enables the support to receive the response payload
+        # for POST and DELETE APIs
+        # See: https://github.com/apache/openwhisk/issues/3274
+        require-response-payload = true
```

Comment:
I looked through the codes.
Since this configuration preserves the current behavior, its validity would be guaranteed by existing test cases.

I think we can add more test cases for the disabled case when we change this configuration to `false` by default.